### ISSUE TITLE
Pages `code` helper throwing after upgrade to v1.0.3

### ISSUE THREAD

**Issue by mbuckley** [allmarkedup/lookbook#133]
Hi! I am attempting to upgrade lookbook from `v0.8.1` to `v1.0.3` and hit an issue where the pages `code` helper is throwing the following error.

```
Wrong number of arguments (given 1, expected 0)
.rbenv/versions/2.6.10/lib/ruby/gems/2.6.0/gems/lookbook-1.0.3/app/helpers/lookbook/component_helper.rb:9:in `code'
```

The piece of code that is failing is

```ruby
# overview.md.erb

<%= code :javascript do %>
 // js code here
<% end %>
```

I've confirmed that it will fail if any `lang` argument is passed when following the docs [here](https://lookbook.build/guide/pages/variables#method-code)

**works**
```ruby
<%= code do %>
  # Ruby code to highlight goes here
<% end %>
```

**fails**
```ruby
<!-- Highlight a different language (HTML) -->
<%= code :html do %>
  <!-- HTML code to highlight goes here -->
<% end %>
```

Ruby version: `2.6.10`

**Comment by allmarkedup** [comment]
Hey @mbuckley thanks for the heads up! I've found the problem and I'm just getting a fix out. I'll let you know when it's done, should be soon.

**Comment by mbuckley** [comment]
Excellent - cheers

**Comment by allmarkedup** [comment]
@mbuckley I've released v1.0.4 which should fix this for you - feel free to reopen if you are still having any problems.

Thanks again for the opening the issue :-)

**Comment by mbuckley** [comment]
@allmarkedup Confirmed that `v1.0.4` fixes the issue! Thanks for the quick turnaround.
